release_notes.md

```diff
@@ -2,16 +2,8 @@
 <!-- Please add your release notes in the following format:
 - My change description (#PR)
 -->
-- Update Python Worker Version to [4.2.0-hotfix](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.2.0-hotfix)
-- Return 500 instead of 403 when writing secrets to storage fails (#8362)
-- Updated Java Worker Version to [2.2.3](https://github.com/Azure/azure-functions-java-worker/releases/tag/2.2.3)
-- Updated Node.js Worker Version to [3.3.0](https://github.com/Azure/azure-functions-nodejs-worker/releases/tag/v3.3.0)
```

Comment:
@Francisco-Gamino and I beat you to the punch in terms of merging a change to the release notes. When you resolve the conflicts you'll want to leave the updated node and powershell bullet points in

node PR: https://github.com/Azure/azure-functions-host/pull/8467
powershell PR: https://github.com/Azure/azure-functions-host/pull/8469

Reply:
Good catch @ejizba , thank you for flagging this.